src/state_control/state_control_Lara.cpp

```diff
@@ -3291,14 +3365,40 @@ int StateControl_LaraDoOneHandWeponFrame(struct entity_s *ent, struct  ss_animat
             case 0: // idle < - > aim;
                 b_tag->is_targeted = (target) ? (0x01) : (0x00);
                 inc_state = Anim_IncTime(ss_anim, (ent->character->state.weapon_ready && do_aim) ? (time) : (-time));
-                if((inc_state == 1) && ent->character->state.weapon_ready && ent->character->cmd.action)
+
+                if ((inc_state == 1) && ent->character->state.weapon_ready && ent->character->cmd.action)
                 {
+                    /// defined if you using windows !
+                    /// in windows the sound is not played the first time (first animation) !
+                    #ifdef _WIN32
```

Comment:
heh... it is hack... really anim command system is wrong. this time I am trying to refactor animation control system (skeletal_model.h) and anim commands call.

Reply:
i will change it, but i cant delete this line in windows project because the sound will be deleted.

Reply:
#CANT_BE_FIXED_FOR_WINDOWS (for now)